client/app/bundles/buttons/button.scss

```diff
@@ -0,0 +1,79 @@
+@import "base.scss";
+
+
+button:hover{
+         
+}
+.buttonFilledM{
+         fontSize: 18px;
+         color: #6D0839;
+         font-weight: bold;
+         background-color: #FFFFFF;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 15px;
+         padding-right: 15px;
+         width: 203px;
+         border-radius: 5px;
+         border-color: #FFFFFF;
+         border-width: 0px;
+         cursor: pointer;
+ }
+
+.buttonGhostM{
+         font-size: 18px;
+         color: #FFFFFF;
+         font-weight : bold;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         width: 128px;
+         border-radius: 5px;
+         opacity: 0.95;
+         border-width: 0px;
+         cursor: pointer;
+      }
+
+.buttonFilled{
+         font-size: 24px;
+         color: #6D0839;
+         font-weight: bold;
+         background-color: #FFFFFF;
+         padding-top: 17px;
+         padding-bottom: 22px;
+         padding-left: 15px;
+         padding-right: 15px;
+         width: 203px;
+         border-radius: 5px;
+         border-color: #FFFFFF;
+         border-width: 0px;
+         cursor: pointer;
+      }
+
+.buttonGhost{
+         font-size: 24px;
+         color: #FFFFFF;
+         font-weight: bold;
+         padding-top: 17px;
+         padding-bottom: 22px;         
+         border-radius: 5px;
+         opacity: 0.95;
+         border-width: 0px;
+         width: 128px;
+         cursor: pointer;
+      }
+
+.buttonGhostXS{
+         font-size: 14px;
+         color: #FFFFFF;
+         font-weight: bold;
+         padding-top: 17px;
+         padding-bottom: 22px;
+         width: 128px;
```

Comment:
Same here for explicit dimensions!

Reply:
Let's make this width a `min-width` instead, and also take out the duplicate width property for this selector!